packages/twenty-front/src/modules/ui/layout/modal/components/ModalLayout.tsx

```diff
@@ -1,21 +1,26 @@
-import React from 'react';
```

Comment:
I have a feeling we don't need to have both `ModalLayout` and `EnhancedModalLayout`, or that I didn't really get your intention. If we think we really do need to have 2 components, then we need to create a story for that new modal component

Reply:
No we dont, thats the whole point.
I am sorry if I was unclear in my comments or description.
I integrated ModalLayout and Modal into EhnhancedModelLayout (naming could be way better)